src/nwb_datajoint/spikesorting/spikesorting_curation.py

```diff
@@ -338,6 +389,17 @@ class MetricSelection(dj.Manual):
     ---
     """
 
+    def insert1(self, key, **kwargs):
```

Comment:
would we never want to compute `peak_offset` on whitened waveform?

Reply:
I don't think so, and certainly we don't now, but of course we can change this later if we want to.